compiler/dl-ir.stanza

```diff
@@ -11,16 +11,32 @@ defpackage stz/dl-ir :
 ;======================== PackageIO =========================
 ;============================================================
 
+public defstruct ImportPrefix :
+  names:False|Tuple<Symbol>
+  prefix:String
+
+public defstruct ImportedPackage <: Hashable & Equalable:
+  package-name: Symbol
+  imported-defs: Tuple<Symbol>|False with: (default => false)
```

Comment:
The `imported-defs`, `prefix` information is information used to calculate forwarding. So change `forward` field to e.g. `False|ForwardingInfo`.